.gitignore

```diff
@@ -104,4 +104,7 @@ dist
 .tern-port
 
 # JS files
-*.js
\ No newline at end of file
+*.js
```

Comment:
On the gitignore, I'd be wary of blanket ignoring `.js` files, and prefer to be more specific, naming either folders or individual files if you have to. The danger is that you'll accidentally ignore something later on and not realise it.